test/test.js

```diff
@@ -292,3 +298,11 @@ test('tagged template non-factory', fileTest);
 test('tagged template factory', fileTest, null, null, factoryHyperConfig);
 
 test('ignore tagged non-function', fileTest, null, true);
+test('lit element partial css', fileTest, null, false, cssLitConfig);
+test('inline css', fileTest, null, false, defaultLitConfig);
+test('link media', fileTest, null, false, defaultLitConfig);
+test('comments', fileTest, null, false, {
```

Comment:
`test('comments', fileTest, null, true, {` - the `true` argument will cause it to use `comments-source.js` as the result.

Reply:
Thanks!